LICENSE.md

```diff
@@ -1,6 +1,6 @@
 # The MIT License (MIT)
 
-Copyright (c) 2014-2019 20tab srl
+Copyright (c) 2014-2022 20tab srl
```

Comment:
```suggestion
Copyright (c) 2014-2024 20tab srl
```

Reply:
I think this change should be applied on both template repositories, I will open a different PR to address it with any other suggested cross template change

Reply:
It would be useful to use a dynamic variable instead of hardcoding the value.